cornell-dti/office-hours
# CSS Fixes for the Queue View

### Inside This PR 


* Increase size of location text input on an existing question
* "I Really Don't Know" dropdown is now layered on top of other components
* If scrolled past the undo banner, banner now sticks to the top
* Default user's image avatar to generic image upon error

### Test Plan 
* Check that display issues are fixed both on desktop and mobile view
* Check that the default user image displays wherever a user avatar is displayed
### Notes 


### Breaking Changes 

- [ ] Database schema change (anything that changes Postgres)
- [ ] I updated existing types in `/src/components/types/`
- [ ] Other change that could cause problems (Detailed in notes)

### Checklist




- [ ] My changes requires a change to the documentation.
